ceph-salt-formula/salt/ceph-salt/time-sync.sls

```diff
@@ -1,5 +1,6 @@
 {% if pillar['ceph-salt']['time_server'].get('enabled', True) %}
 {% set time_server = pillar['ceph-salt']['time_server']['server_host'] %}
+{% set time_server_is_minion = pillar['ceph-salt']['time_server']['is_minion' %}
```

Comment:
Missing a `]` here, although I think we can calculate this on the fly, instead of storing it in the pillar.

I've tested the following:

```diff
+{% set time_server_short_name = pillar['ceph-salt']['time_server']['server_host'].split('.', 1)[0] %}
+{% set time_server_is_minion = time_server_short_name in pillar['ceph-salt']['minions']['all'] %}
+
+{% if time_server_is_minion %}
+is minion - yes:
+  test.nop
+{% else %}
+is minion - no:
+  test.nop
+{% endif %}
```

And it works fine:
```
master:~ # salt node1\* state.show_sls ceph-salt | grep 'is minion'
    is minion - yes:

master:~ # ceph-salt config /time_server/server_hostname set some.host.name
Value set.

master:~ # salt node1\* state.show_sls ceph-salt | grep 'is minion'
    is minion - no:
```